src/java.base/share/classes/sun/net/www/http/ChunkedOutputStream.java

```diff
@@ -59,11 +59,11 @@ public class ChunkedOutputStream extends OutputStream {
     private PrintStream out;
 
     /* the chunk size we use */
-    private int preferredChunkDataSize;
-    private int preferedHeaderSize;
-    private int preferredChunkGrossSize;
+    private final int preferredChunkDataSize;
+    private final int preferedHeaderSize;
```

Comment:
Is it a typo `prefered` ?
Other fields use `preferred` prefix.

Reply:
Hello Andrey, I don't have a strong preference. However, like you note, the other fields are using double `r` and we are anyway in this area changing that code, so I think it's OK to rename this field to use double `r` too. Furthermore, there's no similarly named getter for this field, so changing this field doesn't cause any other impact.